cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/apirequest-count-reader/clusterrolebinding.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

```suggestion
apiVersion: rbac.authorization.k8s.io/v1beta1
apiVersion: rbac.authorization.k8s.io/v1
```

Similar to [Humair's PR](https://github.com/operate-first/apps/pull/1874) rbac.authorization.k8s.io version `v1beta1` has been depricated

---

@Gregory-Pereira Oops missed that, updated